### ISSUE TITLE
[Bug] Modal is not read by screen reader

### ISSUE THREAD

**Issue by abbottmw** [Blazored/Modal#574]
**Modal not being read by screen reader**
I was testing my page for accessibility  and 508 compliance and I noticed using NVDA screen reader ( I don't have JAWS) that it does not read the Modal when it is shown.

Another thing I found is the element that triggered the Modal isn't selected when the Modal closes.

**My tests**
I did some basic tests and compared it with  Blazor Bootstrap Modal.   This Modal seems to read correctly, but it's more of a wrapper to the boostrap library.

Blazored Modal markup is a little different, unless you make your own layout.

Focus trap doesn't seem to work with a custom layout.

I created a [github repo ](https://github.com/abbottmw/Blazor-Modal-Accessibility-Testing) with my testing

I tested with a standard blazor wasm template on .net 8. (Chrome browser)
